pynucastro/templates/amrexastro-cxx-microphysics/table_rates.H.template

```diff
@@ -252,47 +220,47 @@ get_entries(const table_t& table_meta,
         // We're at the first or last table cell (in temperature)
         // First do bilinear interpolation in rhoy for the table at tlo and thi
 
-        Real f_i = bl_clamp(rhoy_lo, rhoy_hi,
+        Real f_i = bl_clamp(log_rhoy_lo, log_rhoy_hi,
                             data(itemp_lo, irhoy_lo, jtab_rate),
                             data(itemp_lo, irhoy_hi, jtab_rate),
-                            rhoy);
+                            log_rhoy);
 
-        Real f_ip1 = bl_clamp(rhoy_lo, rhoy_hi,
+        Real f_ip1 = bl_clamp(log_rhoy_lo, log_rhoy_hi,
                               data(itemp_hi, irhoy_lo, jtab_rate),
                               data(itemp_hi, irhoy_hi, jtab_rate),
-                              rhoy);
+                              log_rhoy);
 
         // Approximate d(rate)/d(t) via forward differencing
 
-        entries(k_drate_dt) = (f_ip1 - f_i) / (temp_hi - temp_lo);
+        entries(k_drate_dt) = (f_ip1 - f_i) / (log_temp_hi - log_temp_lo);
```

Comment:
This should be zero if the temperature is clamped (check `log_temp < log_temp_lo || log_temp > log_temp_hi`)